openlibrary/templates/type/author/view.html

```diff
@@ -168,7 +179,7 @@ <h2 class="collapse">
         </div>
     </div>
     <div class="contentOnethird">
-        <div class="illustration">
+        <div class="illustration" itemprop="image">
```

Comment:
You are correct, I believe this needs to go into `covers/author_photo`. It should go on a `<img>` element.